admin/src/templates/upgrade.html

```diff
@@ -1,35 +1,61 @@
-<div class="col-sm-12" ng-show="loading || upgrading">
-  <div class="loader"></div>
-</div>
 
-<div class="upgrade-config col-sm-12" ng-hide="deployInfo || loading">
+<div class="upgrade-config col-sm-12" ng-hide="currentDeploy || loading">
   <legend translate>instance.upgrade.no_horti</legend>
   <p translate>instance.upgrade.no_horti.detail</p>
 </div>
 
-<div class="upgrade-config col-sm-12" ng-show="deployInfo">
-  <div ng-show="upgrading">
+<div class="upgrade-config col-sm-12" ng-show="currentDeploy">
+  <div ng-show="deployDoc">
+    <legend ng-hide="deployDoc.action === 'stage'" translate>instance.upgrade.deployment</legend>
+    <legend ng-show="deployDoc.action === 'stage'" translate>instance.stage.deployment</legend>
+
     <p class="status"
       translate="instance.upgrade.upgrading"
-      translate-values="{ before: deployInfo.version, after: upgrading }"></p>
+      translate-values="{ before: currentDeploy.version, after: deployDoc.build_info.version }"></p>
+
+    <ul ng-show="deployDoc.log.length" class="table table-striped col">
+      <li ng-repeat="entry in deployDoc.log" class="row">
+        <div ng-show="entry.type === 'stage'">
+          <div class="col-xs-3">{{entry.datetime | date:'medium'}}</div>
+          <div class="col-xs-9">{{entry.message}}</div>
```

Comment:
I disagree here. Do we foresee a close future where the administrators of servers are unfamiliar with english? I ask because IME translations have not been worthwhile in admin situations, and for this to make sense we'd need them to also display correctly in horti's cli output as well.

Reply:
Every other string in the admin interface is translated and that has been our policy for years. I'm willing to have a conversation with stakeholders about whether we want to change our policy on translation in the admin UI, but until then we should stick with the existing policy.

Reply:
I mean it's not really out policy right? Medic-conf isn't translated either, or any of our admin documentation, and those are key to administering our application effectively.

To me It feels like we translate web pages because it's easy to do and we have to do it in the main app so the framework is already there, but we're not actually committed to having all of our tooling available in other languages.